java/.github/workflows/maven.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '8' ]
+        java: [ 17, 21 ]
```

Comment:
For w/e reason, the generator is using later maven version now. Tried both Java 8 and 17 for maven, both working fine locally.